libraries/Microsoft.Bot.Builder/ChannelServiceHandler.cs

```diff
@@ -44,7 +44,7 @@ public class ChannelServiceHandler
         /// <value>
         /// The channel provider that implements <see cref="IChannelProvider"/>.
         /// </value>
-        protected IChannelProvider ChannelProvider => _channelProvider;
+        protected IChannelProvider ChannelProvider { get; }
```

Comment:
why are we making this protected? going forward we will be deprecating this as it only exists to support the old ChannelService concept.

Reply:
I John, this is already protected in the product today, I just removed the instance variable (small cleanup), we will need to deprecate this property (or do a small breaking change and remove it) once we have the new auth classes.